packages/expect/src/index.ts

```diff
@@ -322,7 +324,7 @@ const makeThrowingMatcher = (
           .then(aResult => processResult(aResult, asyncError))
           .catch(error => handlError(error));
       } else {
-        const syncResult = ((potentialResult: any): SyncExpectationResult);
+        const syncResult: SyncExpectationResult = potentialResult as any;
```

Comment:
```suggestion
        const syncResult = potentialResult as SyncExpectationResult;
```

Reply:
Nice catch. I tried to stick as closely as possible to the flow type defs.